src/main/java/core/basesyntax/figures/Square.java

```diff
@@ -0,0 +1,24 @@
+package core.basesyntax.figures;
+
+public class Square extends Figure {
+    private int side;
```

Comment:
better to use double. Same in another cases. you also need to use nextDouble instead of nextInt